.eslintrc.js

```diff
@@ -39,11 +45,10 @@ module.exports = {
     '@typescript-eslint/generic-type-naming': 'off',
     indent: 'off',
     '@typescript-eslint/indent': 'off',
-    '@typescript-eslint/interface-name-prefix': ['error', 'never'],
```

Comment:
Why remove this rule? I like this one. It's not needed to prefix interfaces with `I` in typescript.

Reply:
it is removed in newest version